main/src/library/Path.flix

```diff
@@ -126,6 +129,26 @@ namespace Path {
         javaFile.canWrite()
     })
 
+    ///
+    /// Read the specified path into an array of bytes.
+    ///
+    pub def readBuffer(p: Path): Result[Array[Int8], IOError] & Impure = try {
+        use Core/Io/InputStream.{available, readBuffer};
+        use Core/Io/InputStream.InputStream.InputStream;
+        import new java.io.FileInputStream(##java.io.File) as newFileInputStream;
+        let Path(javaFile) = p;
+        let javaInputStream = newFileInputStream(javaFile);
+        let inputStream = InputStream(javaInputStream as ##java.io.InputStream);
+        inputStream.available()
```

Comment:
Maybe I am confused, but I am not sure if this is correct.

`available` does not return the length of an input stream, only what can currently be read.

Without iteration, I don't see how this is reading the entire input stream.